account_payment_partner/README.rst

```diff
@@ -111,6 +111,9 @@ Contributors
 
   * Pedro M. Baeza <pedro.baeza@tecnativa.com>
   * Carlos Dauden <carlos.dauden@tecnativa.com>
+* `DynApps <https://www.dynapps.be>`_:
```

Comment:
Leave an empty line before

Reply:
![image](https://user-images.githubusercontent.com/8638295/61223395-d3739100-a71c-11e9-8526-7571e7f53293.png)
Before is not needed?

Reply:
OK, I see... I though it was, sorry.

Reply:
It has no influence on the result...